### PR TITLE
#3110 Adding missing translations in landing in en.json

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

### What's new?

- [x] PR related #3110  
- [ ] Requires deployment <rubick/magick_issue>
- [ ] <brief_description_of_what_I've_added>

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

File: components/rmrk/Gallery/LatestSales.vue [hunk 1]
```diff
@@ -5,7 +5,9 @@
     <div class="columns is-vcentered">
       <div class="column is-four-fifths">
         <h1 class="title is-2">{{ $t('general.latestSales') }}</h1>
-        <p class="subtitle is-size-5">Discover the most recent sales on rmrk</p>
+        <p class="subtitle is-size-5">
+          {{ $t('landingPage.latestSalesheading') }}
```
Comment (roiLeo) [inline components/rmrk/Gallery/LatestSales.vue:9]: `general.latestSalesSubheading`

File: components/rmrk/Gallery/NewestList.vue [hunk 2]
```diff
@@ -3,8 +3,8 @@
     <Loader v-model="isLoading" />
     <div class="columns is-vcentered">
       <div class="column is-four-fifths">
-        <h1 class="title is-2">Newest List</h1>
-        <p class="subtitle is-size-5">Discover the latest items on sale</p>
+        <h1 class="title is-2">{{ $t('landingPage.newestListHeading') }}</h1>
```
Comment (roiLeo) [inline components/rmrk/Gallery/NewestList.vue:6]: `general.newestList`

File: components/rmrk/Gallery/NewestList.vue [hunk 3]
```diff
@@ -3,8 +3,8 @@
...
+        <p class="subtitle is-size-5">{{ $t('landingPage.newestListDesc') }}</p>
```
Comment (roiLeo) [inline components/rmrk/Gallery/NewestList.vue:7]: `general.newestListSubheading`

File: components/rmrk/Gallery/PopularCollections.vue [hunk 4]
```diff
@@ -4,9 +4,11 @@
 
     <div class="columns is-vcentered">
       <div class="column is-four-fifths">
-        <h1 class="title is-2">{{ $t('Popular Collections') }}</h1>
+        <h1 class="title is-2">
+          {{ $t('landingPage.popularCollectionsHeading') }}
```
Comment (roiLeo) [inline components/rmrk/Gallery/PopularCollections.vue:8]: `general.popularCollections`

File: components/rmrk/Gallery/PopularCollections.vue [hunk 5]
```diff
@@ -4,9 +4,11 @@
...
+        </h1>
         <p class="subtitle is-size-5">
-          Discover the most popular collections on rmrk
+          {{ $t('landingPage.popularCollectionsDesc') }}
```
Comment (roiLeo) [inline components/rmrk/Gallery/PopularCollections.vue:11]: `general.popularCollectionsSubheading`

File: components/landing/Landing.vue [hunk 6]
```diff
@@ -17,10 +17,10 @@
           <h3
             class="title is-1 is-size-3-mobile is-flex is-flex-direction-column uppercase homepage__heading">
             <span class="text__stroked my-5">
-              Discover, collect and sell Kusama NFTs
+              {{ $t('landingPage.ksmDiscoverText') }}
             </span>
             <span class="title is-6 homepage__subtitle uppercase text-semibold">
-              Built on
+              {{ $t('landingPage.ksmBuilt') }}
```
Comment (roiLeo) [inline components/landing/Landing.vue:23]: `helper.builtOn`

File: components/landing/Landing.vue [hunk 7]
```diff
@@ -17,10 +17,10 @@
           <h3
             class="title is-1 is-size-3-mobile is-flex is-flex-direction-column uppercase homepage__heading">
             <span class="text__stroked my-5">
-              Discover, collect and sell Kusama NFTs
+              {{ $t('landingPage.ksmDiscoverText') }}
```
Comment (roiLeo) [inline components/landing/Landing.vue:20]: `general.homeHeading`

**Review by roiLeo — COMMENTED** [review]
Hey, I would do something more like this.

I think some translations are still missing but it's already a good start!

File: components/landing/Landing.vue [hunk 8]
```diff
@@ -17,10 +17,10 @@
           <h3
             class="title is-1 is-size-3-mobile is-flex is-flex-direction-column uppercase homepage__heading">
             <span class="text__stroked my-5">
-              Discover, collect and sell Kusama NFTs
+              {{ $t('general.ksmDiscoverText') }}
             </span>
             <span class="title is-6 homepage__subtitle uppercase text-semibold">
-              Built on
+              {{ $t('helper.ksmBuilt') }}
```
Comment (roiLeo) [inline components/landing/Landing.vue:23]:
```suggestion
              {{ $t('helper.builtOn') }}
```

File: langDir/en.json [hunk 9]
```diff
@@ -630,7 +637,8 @@
       "label": "URL to transform",
       "placeholder": "singular.rmrk.app/collectibles/",
       "copy": "Kodadot link successfuly copied"
-    }
+    },
+    "ksmBuilt": "Built on"
```
Comment (roiLeo) [inline langDir/en.json:641]:
```suggestion
    "builtOn": "Built on"
```

**Review by roiLeo — APPROVED** [review]
✅ lgtm

**Comment by yangwao** [comment]
pay 100 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $100 @ 66.74 USD/KSM ~ 1.498 $KSM
🧗 [EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU](https://kusama.subscan.io/account/EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)
🔗 [0x8284d0fae1e134c188987e2cc95d5db2a6ec88925772ea9f0b055b2994399f78](https://kusama.subscan.io/extrinsic/0x8284d0fae1e134c188987e2cc95d5db2a6ec88925772ea9f0b055b2994399f78)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**